src/utils.js

```diff
@@ -469,7 +469,12 @@ function createKeyValueObjectFromArray (inputsArray = []) {
     if (input.key && input.value) {
       try {
         // assume it is JSON, there is only 1 way to find out
-        tempObj[input.key] = JSON.parse(input.value)
+        if (typeof input.value === 'string' && ['{', '['].indexOf(input.value.charAt(0)) > -1) {
```

Comment:
@purplecabbage Perhaps it is just me, but the nesting here seems off. In pseudo-code, i would expect..

```
if (looksLikeJson(input.value) {
  try {
     value = JSON.parse(input.value)
   } catch (e) {
      log(e)
      value = input.value
   }
} else {
  value = input.value
}
```

In other words, the try/catch should be inside the if -- whereas you have it outside.